daml/Marketplace/Trading/Order.daml

```diff
@@ -44,12 +45,10 @@ data Status
     | PartiallyExecuted
     | FullyExecuted
     | Rejected with
-        errorCode : Int
-        errorMessage : Text
+        reason : Error.T
     | PendingCancellation
     | CancellationRejected with
-        errorCode : Int
-        errorMessage : Text
+        reason : Error.T
```

Comment:
:+1: Think I mentioned this before.

Reply:
Yup, this is based off your feedback from before - I just hadn't gotten around to updating the `Order`